collect-logs

```diff
@@ -209,12 +235,20 @@ def juju_status(juju):
     output = output.decode("utf-8").strip()
     return yaml.load(output)
 
+def get_bootstrap_ip(juju, status=None):
+    if status is None:
+        status = juju_status(juju)
+    if "machines" not in status:
+        sys.exit("ERROR, no machines found. Make sure the right juju "
+                 "environment is set.")
+    return status["machines"]["0"]["dns-name"]
```

Comment:
Agreed, I couldn't decide which to pull and wasn't certain when the "ip-addresses" list got longer than 1.